### PR TITLE
Adding resetToRoute API

### PR DESCRIPTION
**Pull request by davidLeonardi**
This API resets the current stack to 0, and thus can be used in deeper routes to begin again with a brand new navigator stack.

I created it so that replaceRoute can be used to replace the current route and maintain the previous navigator stack, and resetToRoute can be used to replace the current route and also start over with a blank navigator stack. 

This means that "back" navigations will be gone.

### CODE REVIEW

**Comment by MikaelCarpenter** [comment]
Awesome! Can you update the README to document the new feature you added. Also check out a previous [PR](https://github.com/MikaelCarpenter/gb-native-router/pull/26/files) that got rid of the drag functions. This branch still has them, so if you could just get this one up to date then its good to go! (:

**Comment by davidLeonardi** [comment]
OK for the readme, didnt quite understand what you meant about the other PR..
Not sure i see the overlap. 
Or am i missing something?

**Comment by davidLeonardi** [comment]
By the way, a release to npm with this and the version bump would be awesome :)

File: index.js [hunk 1]
```diff
@@ -94,14 +91,35 @@ var Router = React.createClass({
...
 
-    var setTitleProps = function(props) {
-      this.setState({ titleProps: props });
-    }.bind(this);
-
     var customAction = function(opts) {
       this.customAction(opts);
     }.bind(this);
 
+    var didStartDrag = function(evt) {
```
Comment (MikaelCarpenter) [inline index.js]: This segment right here was removed in the PR

**Comment by MikaelCarpenter** [comment]
but yea once we get these in I'll release to npm :) I've been having issues with npm lately that i'm still trying to resolve

**Comment by MikaelCarpenter** [comment]
looks great man! thanks for the contribution :+1:

**Pull request merged**